charts/invenio/values.yaml

```diff
@@ -99,12 +99,11 @@ invenio:
   ## @param invenio.extraConfig Extra environment variables (templated) to be added to all the pods.
   ##
   extraConfig: {}
+  ## @param invenio.extra_env_from_secret DEPRECATED: invenio.extraEnvFromSecret instead
   extra_env_from_secret: []
-    # - name: NAME_OF_MY_ENV_VAR
-    #   valueFrom:
-    #     secretKeyRef:
-    #       name: name-of-my-secret
-    #       key: KEY_IN_MY_SECRET
+  ## @param invenio.extraEnvVars Extra environment variables (templated) to be added to all the pods.
```

Comment:
> templated

Are they? 🤔

Reply:
None of them are actually, not even the existing ones 😂 
Should I update the docs everywhere or add `tpl` instead? 🤔

Reply:
Good question. I lean towards just updating the comments/docs, because adding `tpl` adds complexity and I prefer doing that only when  there's a clear reason to do so. But I'm open to either alternative.